SimpleAkkaDotNetCluster/SimpleClusterCaller/Program.cs

```diff
@@ -18,46 +16,28 @@
     var _baseUrl2 = "akka.tcp://ClusterSystem@localhost:2552";
 
     var thisPort = 3000;
-    var baseLocation = Assembly.GetAssembly(typeof(Program));
-    var dirInfo = new DirectoryInfo(baseLocation.Location);
-    var hoconFilePath = dirInfo.Parent + "\\akka-hocon.conf";
-    Console.WriteLine(hoconFilePath);
-    var hoconFile = XElement.Parse(File.ReadAllText(hoconFilePath));
-
-    var config = ConfigurationFactory.ParseString(hoconFile.Descendants("hocon").Single().Value);
+    var hoconFile = File.ReadAllText("akka-hocon.conf");
+    var config = ConfigurationFactory.ParseString(hoconFile);
 
     var clusterNodeConfig =
                         ConfigurationFactory.ParseString("akka.remote.dot-netty.tcp.port=" + thisPort)
                             .WithFallback(config);
 
     //create an Akka system
     Console.WriteLine("Create actor system");
-    var actorSystem = ActorSystem.Create("ClusterSystem", clusterNodeConfig);
+    var actorSystem = ActorSystem.Create("ClusterClientSystem", clusterNodeConfig);
     Console.WriteLine("Actor system created");
 
-    var intialContacts = ImmutableHashSet<ActorPath>.Empty
-                .Add(ActorPath.Parse(_baseUrl + "/system/receptionist"))
-                .Add(ActorPath.Parse(_baseUrl2 + "/system/receptionist"));
-
-    var clusterClientSettings = ClusterClientSettings.Create(actorSystem)
-        .WithInitialContacts(intialContacts);
-
-
-    var clusterClientProps = ClusterClient.Props(clusterClientSettings);
+    var initialContacts = ImmutableHashSet<ActorPath>.Empty
+                .Add(ActorPath.Parse(_baseUrl) / "system" / "receptionist")
```

Comment:
The "/" operator here is not a division, it is a syntactic sugar to construct actor paths.